modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/PodcastFragment.kt

```diff
@@ -625,6 +635,13 @@ class PodcastFragment : BaseFragment(), Toolbar.OnMenuItemClickListener, Corouti
             }
         )
 
+        viewModel.ratingsState.observe(viewLifecycleOwner) { state ->
+            when (state) {
+                is PodcastViewModel.RatingState.Loaded -> adapter?.setRatings(state.ratings)
+                is PodcastViewModel.RatingState.Error -> Unit // Do Nothing
```

Comment:
Depending on how the UI feature for this plays out, I wonder if it might make sense to include all the ratings retrieval logic ina new dedicated ViewModel inside the `StarRatingView` (or maybe a wrapper composable that handles the `StarRatingView` and whatever loading/error UI we want to present). 

I wouldn't worry about changing this now since depending on the UI requirements that change might not make sense. But it may be worth keeping in mind.